.eslintrc.yaml

```diff
@@ -65,6 +65,7 @@ rules:
       pattern:
         vue: always
   import/newline-after-import: off # prevent collision with semi-style: first
+  import/prefer-default-export: off
```

Comment:
question: why do we disable this?

Reply:
Named exports are better, we only need default exports in Vue SFC.